src/detect-transform-base64.c

```diff
@@ -37,33 +37,48 @@
 #include "util-unittest.h"
 #include "util-print.h"
 
-static int DetectTransformFromBase64DecodeSetup(DetectEngineCtx *, Signature *, const char *);
-static void DetectTransformFromBase64DecodeFree(DetectEngineCtx *, void *);
 #ifdef UNITTESTS
 #define DETECT_TRANSFORM_FROM_BASE64_MODE_DEFAULT (uint8_t) SCBase64ModeRFC4648
 static void DetectTransformFromBase64DecodeRegisterTests(void);
 #endif
-static void TransformFromBase64Decode(
-        DetectEngineThreadCtx *det_ctx, InspectionBuffer *buffer, void *options);
+#define B64D_ID_DATA_LEN 128
```

Comment:
Could you comment on it ? How did you compute it ?

Reply:
Could you clarify the question -- are you asking why did I choose 128 to represent the identity string?

Reply:
That was the intent

Reply:
It is not obvious to me looking at the string that it can be at most 128 chars. How did you compute it ?

Reply:
The idea is that the string will not be larger than 128 bytes ... it can be shorter, and the hash function only needs to know the bytes and the byte count.

I manually counted an example string and rounded up to get 128.

Reply:
Looks dangerous if the structure and format string grows...